kube-flannel.yml

```diff
@@ -32,7 +100,121 @@ data:
       "Network": "10.244.0.0/16",
       "Backend": {
         "Type": "vxlan",
-        "VNI": 4096,
+        "VNI" : 4096,
```

Comment:
why the space?

Reply:
Probably some IDE formatting. I'll get rid of this